main/reference/ertp-api/ertp-data-types.md

```diff
@@ -50,8 +50,8 @@ There are several kinds of Assets.
 
 - **AssetKind.NAT** : Used with fungible assets. **AmountValues** are natural numbers using the JavaScript  [BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) type to avoid overflow risks from using the usual JavaScript **Number** type.
 - **AssetKind.SET** : Deprecated.
-- **AssetKind.COPY_SET** : Used with non-fungible assets where there can't be duplicates (e.g., assigned seats in a stadium). **AmountValues** are arrays of objects.
-- **AssetKind.COPY_BAG** : Used with non-fungible assets where there can be duplicates. (e.g., weapons in a computer game). **AmountValues** are arrays of objects.
+- **AssetKind.COPY_SET** : Used with non-fungible assets where there can't be duplicates (e.g., assigned seats in a stadium). See more [here](https://endojs.github.io/endo/types/_endo_patterns.CopySet.html).
```

Comment:
no _here_ links, please.

same rationale as https://github.com/Agoric/documentation/pull/1104#discussion_r1624864872 etc.

Reply:
@dckc Thanks for pointing this out!